app/services/explorer.js

```diff
@@ -867,30 +874,35 @@ export default Ember.Service.extend({
      *    [{"name":"customers","n_val":3,"schema":"_yz_default"}]
      */
     getIndexes(clusterId) {
-        var url = this.getClusterProxyUrl(clusterId) + '/search/index';
+        var url = `${this.getClusterProxyUrl(clusterId)}/search/index`;
 
-        var request = new Ember.RSVP.Promise(function(resolve, reject) {
-            Ember.$.ajax({
-                type: "GET",
-                url: url
-            }).then(
-                // Success
-                function(data) {
-                    resolve(data);
-                },
-                // Error
-                function(jqXHR, textStatus) {
-                    if(jqXHR.status === 404) {
-                        // No indexes found, simply return an empty list
-                        resolve([]);
```

Comment:
Getting rid of this line (of treating a 404 result to getIndexes as a success with value `[]`) is going to break other routes. 
While this is functionally the same as not creating any records in the Clusters route, that's only the case because Clusters traps failed promises via `allSettled` in its model function. The other routes don't use this mechanism, and I think will error out instead.